ingeniamotion/enums/__init__.py

```diff
@@ -1,9 +1,21 @@
 from enum import Enum, IntEnum
+from typing import Type, TypeVar
 
 from ingenialink.canopen.network import CAN_BAUDRATE, CAN_DEVICE
 from ingenialink.enums.register import REG_ACCESS, REG_DTYPE
 
+T = TypeVar("T", bound=Type[Enum])
 
+__all__ = ["CAN_BAUDRATE", "CAN_DEVICE", "REG_ACCESS", "REG_DTYPE"]
+
+
+def export(obj: T) -> T:
+    """Decorator use to explicitly export a class"""
```

Comment:
If it's T is bound to enum only works for enums, not classes

Reply:
I know. Only enums should be defined in this module.